exporter/clickhouseexporter/README.md

```diff
@@ -0,0 +1,90 @@
+# ClickHouse Exporter
+
+**Status: experimental**
+Supported pipeline types: logs.
+
+This exporter supports sending OpenTelemetry logs to [ClickHouse](https://clickhouse.com/).
+
+Note:
+Always add [batchprocessor](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor) to collector pipeline, as Clickhouse [recommend](https://clickhouse.com/docs/en/introduction/performance/#performance-when-inserting-data ) inserting data in packets of at least 1000 rows, or no more than a single request per second.
```

Comment:
Just something to think about, not a recommendation: it's possible to use [Buffer engine](https://clickhouse.com/docs/en/engines/table-engines/special/buffer/) for this purpose. But it has its own drawbacks.

Reply:
Buffer is awsome feature, but it can not replace the batch-processor. 
> Note that it does not make sense to insert data one row at a time, even for Buffer tables. This will only produce a speed of a few thousand rows per second, while inserting larger blocks of data can produce over a million rows per second (see the section “Performance”).

the `Performance` section clickhouse document says that 
> We recommend inserting data in packets of at least 1000 rows, or no more than a single request per second. 

https://clickhouse.com/docs/en/introduction/performance/#performance-when-inserting-data.

Reply:
I have update README.md just now.